examples/3D-bf-to-2D-phase.py

```diff
@@ -48,15 +47,19 @@
 print(f"Shape of 2D phase data: {np.shape(phase2D)}")
 
 ## Save to zarr
-writer = WaveorderWriter("./output")
-writer.create_zarr_root("reconstructions_" + timestamp)
-writer.init_array(
-    position=0,
-    data_shape=(1, 1, 1, Y, X),
-    chunk_size=(1, 1, 1, Y, X),
-    chan_names=["Phase"],
+dataset = open_ome_zarr(
+    "./output/reconstructions_" + timestamp,
+    layout="fov",
+    mode="w",
+    channel_names=["Phase"],
 )
-writer.write(phase2D, p=0, t=0, c=0, z=0)
+img = dataset.create_zeros(
+    name="0",
+    shape=(1, 1, 1, Y, X),
+    dtype=phase2D.dtype,
+    chunks=(1, 1, 1, Y, X),  # chunk by XY planes
+)
+img[0, 0, 0] = phase2D
```

Comment:
If we only want to show how to write a single slice, it can be:
```suggestion
dataset["0"] = phase2D[np.newaxis, np.newaxis, ...]
```
But it is also nice to show that this is potentially a time series.

Reply:
Very nice!

If you use `dataset["0"] = phase2D[np.newaxis, np.newaxis, ...]`, I'm guessing that it will use the default chunk size? Do you need to use `create_zeros` or `create_image` if you want to set the chunk size?

Just to make sure, is `dataset["0"] = phase2D[np.newaxis, np.newaxis, np.newaxis, ...]` (three new axes) the equivalent code? Or is there something I don't understand?

Reply:
> I'm guessing that it will use the default chunk size? Do you need to use `create_zeros` or `create_image` if you want to set the chunk size?

Yes, in this case they are equivalent since the default chunk size is ZYX.

> Just to make sure, is `dataset["0"] = phase2D[np.newaxis, np.newaxis, np.newaxis, ...]` (three new axes) the equivalent code? Or is there something I don't understand?

Sorry that was a typo, you are right that there should be three new axes.